qemu/tests/cfg/migration_with_json_backing.cfg

```diff
@@ -0,0 +1,23 @@
+- migration_with_json_backing:
+    virt_test_type = qemu
+    type = migration_with_json_backing
+    kill_vm = yes
+    start_vm = no
+    mig_timeout = 3600
+    pre_command = "sync && echo 3 > /proc/sys/vm/drop_caches"
+    image_chain = "image1 sn1 sn2 sn3"
+    qcow2:
+        image_format_image1 = qcow2
+    raw:
+        image_format_image1 = raw
+    luks:
+        image_format_image1 = luks
```

Comment:
we have to accept the risk that if another format other than qcow2/raw/luks is used, we have to add it in cfg, or we'll hit an error. Or we use only qcow2 raw luks to limit the image format

Reply:
And do we need to add image_name_image1 = ${image_name} ?

Reply:
@zhencliu, when we set  image_name_image1 = ${image_name}  here, test will fail with make_create_command() error. So I think it can't analyze the ${image_name} here.

Reply:
Thanks for the information. iamge_name_image1 = image, which is originally defined in base.cfg